transformations/index.md

```diff
@@ -371,3 +377,9 @@ everything is less clear and isolated. To save time, run multiple orchestration
 #### Dependencies
 Dependencies allow you to chain transformation steps in older projects. A given transformation is executed after all 
 required steps have been executed.
+
+## Read-only Input mapping
+
+With Read-only input-mapping feature you can access all buckets (your own or linked) in transformations. Your transformation user
+has read only access on buckets (and their tables) so you can access such data. So there is no need to specify standard input mapping 
+for your transformations. Name of backend object (database, schema...) depends on the backend you use, and it contains bucket ID (not bucket name). 

```

Comment:
Not really :( https://github.com/keboola/connection/blob/69877c0000f272dd69ab8896f478fb8cad1a059f/legacy-app/application/modules/storage/services/BucketBackend/Snowflake.php#L142

Reply:
V SNFLK máš:

"in.c-bucket"."tableMyTable"

Jde tam stringId()

Reply:
vyreseno osobne